<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParcelInfoFeeDialog</class>
 <widget class="QDialog" name="ParcelInfoFeeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>867</width>
    <height>537</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>867</width>
    <height>537</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>880</width>
    <height>550</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>867</width>
    <height>537</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="close_button">
   <property name="geometry">
    <rect>
     <x>789</x>
     <y>510</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Хаах</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>2</y>
     <width>861</width>
     <height>503</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Fee</string>
    </attribute>
    <widget class="QGroupBox" name="find_gbox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>0</y>
       <width>844</width>
       <height>39</height>
      </rect>
     </property>
     <property name="title">
      <string>Find</string>
     </property>
     <widget class="QPushButton" name="find_button">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Find</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/plugins/lm2/zoom.png</normaloff>:/plugins/lm2/zoom.png</iconset>
      </property>
     </widget>
     <widget class="QLineEdit" name="old_parcel_id_edit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>11</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>13</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Регистр:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>165</x>
        <y>13</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Хуучин НТ Дугаар:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>405</x>
        <y>12</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Гэрээний дугаар:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="contract_no_edit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>10</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="person_id_edit">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="clear_button">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>New</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>250</y>
       <width>281</width>
       <height>225</height>
      </rect>
     </property>
     <property name="title">
      <string>Ногдуулалт, төлбөл зохих төлбөр</string>
     </property>
     <widget class="QDoubleSpinBox" name="payment_total_sbox">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>125</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 170, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>105</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 170, 255);</string>
      </property>
      <property name="text">
       <string>Нийт төлбөр</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>15</y>
        <width>97</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Гэрээ ногд. Төлбөр</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="payment_contract_sbox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>35</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>87</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Дутуу Төлөлт(+)</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="payment_before_less_sbox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="payment_before_over_sbox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>125</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>105</y>
        <width>74</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Илүү Төлөлт(-)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_29">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>128</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Тухайн онд ногд. Төлбөр</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="payment_year_sbox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_30">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>60</y>
        <width>97</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Алданги, торгууль</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="payment_fund_sbox">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>35</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="payment_loss_sbox">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_31">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>15</y>
        <width>70</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Нөхөн төлбөр</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>250</y>
       <width>281</width>
       <height>224</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(233, 247, 255);</string>
     </property>
     <property name="title">
      <string>Төлөлт</string>
     </property>
     <widget class="QDoubleSpinBox" name="paid_less_sbox">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>125</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 170, 127);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_39">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>105</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 170, 127);</string>
      </property>
      <property name="text">
       <string>Дутуу төлөлт</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_40">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>15</y>
        <width>99</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Урд оны дутуугаас</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="paid_before_less_sbox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>35</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_41">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>132</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Тайлангаар ногдуулснаас</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="paid_year_sbox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="paid_fund_sbox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>125</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_42">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>105</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Хүү, торгууль, бусад</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_43">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Дүүрэгт төлсөн</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="paid_district_sbox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_44">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>60</y>
        <width>104</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Хүчингүй болсон дүн</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="paid_city_sbox">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>35</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="paid_invalid_sbox">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_45">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>15</y>
        <width>86</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Нийслэлд төлсөн</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_46">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>150</y>
        <width>62</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Илүү төлөлт</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="paid_over_sbox">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>170</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>261</width>
       <height>226</height>
      </rect>
     </property>
     <property name="title">
      <string>Мэдээлэл</string>
     </property>
     <widget class="QLabel" name="label_47">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>16</y>
        <width>97</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Гэрээний дугаар</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="payment_contract_edit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>36</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Тайлбар</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="decsription_txt">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>241</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_48">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>97</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Зориулалт</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="payment_zoriulalt_edit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>241</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="payment_name_edit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>126</y>
        <width>241</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_49">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>106</y>
        <width>97</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Оноосон нэр</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_50">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>16</y>
        <width>97</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Талайн хэмжээ /м2/</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="payment_area_edit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>36</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>234</y>
       <width>841</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="lineWidth">
      <number>5</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>68</y>
       <width>851</width>
       <height>171</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Төлбөрийн мэдээлэл</string>
      </attribute>
      <widget class="QTableWidget" name="payment_twidget">
       <property name="geometry">
        <rect>
         <x>4</x>
         <y>3</y>
         <width>838</width>
         <height>141</height>
        </rect>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <column>
        <property name="text">
         <string>Төлөв</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Төрөл</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>НТ-н дугаар</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Жил</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Регистр</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Гэрээний дугаар</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Гэрээнд ногдуулсан дүн</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Тухайн онд ногдуулсан дүн</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Тухайн жил нийт</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Урд оны дутуу төлөлтөөс</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Урд оны илүү төлөлтөөс</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Нийслэлд төлсөн</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Дүүрэгт төлсөн</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Хүчингүйд тооцсон дүн</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Тайлан оны илүү, дутуу төлөлт </string>
        </property>
       </column>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>ЛМ-н төлбөр төлөлтийн мэдээлэл</string>
      </attribute>
      <widget class="QTableWidget" name="paid_twidget">
       <property name="geometry">
        <rect>
         <x>4</x>
         <y>3</y>
         <width>838</width>
         <height>110</height>
        </rect>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <column>
        <property name="text">
         <string>Жил</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Регистр</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Гэрээний дугаар</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Гэрээнд ногдуулсан дүн</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Тухайн онд ногдуулсан дүн</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Тухайн жил нийт</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Урд оны дутуу төлөлтөөс</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Бүгд дүн</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Хүчингүйд тооцсон дүн</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Тайлан оны илүү, дутуу төлөлт </string>
        </property>
       </column>
      </widget>
     </widget>
    </widget>
    <widget class="QComboBox" name="object_cbox">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>43</y>
       <width>401</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="year_cbox">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>43</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="city_type_cbox">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>43</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="save_button">
   <property name="geometry">
    <rect>
     <x>704</x>
     <y>510</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Хадгалах</string>
   </property>
  </widget>
  <widget class="QPushButton" name="finish_button">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>510</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Дуусгах</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="one_select_chbox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>510</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Зөвхөн сонгогдсон жилийг оруулах</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="status_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_button">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>510</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Устгах</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>old_parcel_id_edit</tabstop>
  <tabstop>find_button</tabstop>
  <tabstop>person_id_edit</tabstop>
  <tabstop>one_select_chbox</tabstop>
  <tabstop>delete_button</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>payment_total_sbox</tabstop>
  <tabstop>finish_button</tabstop>
  <tabstop>save_button</tabstop>
  <tabstop>clear_button</tabstop>
  <tabstop>contract_no_edit</tabstop>
  <tabstop>payment_contract_sbox</tabstop>
  <tabstop>payment_before_less_sbox</tabstop>
  <tabstop>payment_before_over_sbox</tabstop>
  <tabstop>payment_year_sbox</tabstop>
  <tabstop>payment_fund_sbox</tabstop>
  <tabstop>payment_loss_sbox</tabstop>
  <tabstop>paid_less_sbox</tabstop>
  <tabstop>paid_before_less_sbox</tabstop>
  <tabstop>paid_year_sbox</tabstop>
  <tabstop>paid_fund_sbox</tabstop>
  <tabstop>paid_district_sbox</tabstop>
  <tabstop>paid_city_sbox</tabstop>
  <tabstop>paid_invalid_sbox</tabstop>
  <tabstop>paid_over_sbox</tabstop>
  <tabstop>payment_contract_edit</tabstop>
  <tabstop>decsription_txt</tabstop>
  <tabstop>payment_zoriulalt_edit</tabstop>
  <tabstop>payment_name_edit</tabstop>
  <tabstop>payment_area_edit</tabstop>
  <tabstop>tabWidget_2</tabstop>
  <tabstop>payment_twidget</tabstop>
  <tabstop>paid_twidget</tabstop>
  <tabstop>object_cbox</tabstop>
  <tabstop>year_cbox</tabstop>
  <tabstop>city_type_cbox</tabstop>
  <tabstop>close_button</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
